.gitignore

```diff
@@ -1,6 +1,8 @@
 *.deb
 *.swp
-carbon-relay-ng
-carbon-relay-ng.test
+/carbon-relay-ng
```

Comment:
`carbon-relay-ng` matched anywhere since it wasn't anchored, so when I changed `cmd/carbon-relay-ng/carbon-relay-ng.go` I got a notice about the folder matching the ignore rule